frontend/src/Card.jsx

```diff
@@ -66,7 +66,7 @@ const Card = ({id, title, text, img, upvotes, author, pinnedBool}) => {
         <div className="card">
             <h3>{title}</h3>
             <p>{text}</p>
-            <img src={img} alt="card img"/>
+            <img src={img} alt="card img" onClick={() => openViewCardModal(id)}/>
```

Comment:
I find it a little confusing that the image of the kudo post is clickable but the image of the kudo board is not.